mods/d2k/maps/harkonnen-08/harkonnen08.lua

```diff
@@ -0,0 +1,307 @@
+AtreidesBase = { AConYard, AOutpost, ARefinery1, ARefinery2, AHeavyFactory, ALightFactory, AHiTechFactory, ARepair, AStarport, AGunt1, AGunt2, ARock1, ARock2, APower1, APower2, APower3, APower4, APower5, APower6, APower7, APower8, APower9 }
+MercenaryBase = { MHeavyFactory, MStarport, MGunt, MPower1, MPower2 }
+
+OrdosReinforcements =
+{
+	easy =
+	{
+		{ "trooper", "trooper", "quad", "quad" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf" },
+		{ "light_inf", "light_inf", "light_inf", "raider", "raider" },
+		{ "combat_tank_o", "quad" }
+	},
+
+	normal =
+	{
+		{ "trooper", "trooper", "trooper", "quad", "quad" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "raider", "raider" },
+		{ "combat_tank_o", "combat_tank_o" },
+		{ "raider", "raider", "quad", "quad", "deviator" }
+	},
+
+	hard =
+	{
+		{ "trooper", "trooper", "trooper", "trooper", "quad", "quad" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "light_inf" },
+		{ "light_inf", "light_inf", "light_inf", "light_inf", "light_inf", "raider", "raider" },
+		{ "combat_tank_o", "combat_tank_o", "quad" },
+		{ "raider", "raider", "raider", "quad", "quad", "deviator" },
+		{ "siege_tank", "combat_tank_o", "combat_tank_o", "raider" }
+	}
+}
+
+MercenaryStarportReinforcements =
+{
+	{ "trooper", "trooper", "trooper", "trooper", "trooper", "trooper", "quad", "quad" },
+	{ "quad", "combat_tank_o", "trike", "quad", "trooper", "trooper" },
+	{ "trooper", "trooper", "trooper", "trooper", "siege_tank", "siege_tank" },
+	{ "quad", "quad", "combat_tank_o", "combat_tank_o", "combat_tank_o" }
+}
+
+OrdosAttackDelay =
+{
+	easy = DateTime.Minutes(3),
+	normal = DateTime.Minutes(2) + DateTime.Seconds(20),
+	hard = DateTime.Minutes(1)
+}
+
+MercenaryStarportDelay = DateTime.Minutes(1) + DateTime.Seconds(20)
+
+OrdosAttackWaves =
+{
+	easy = 4,
+	normal = 5,
+	hard = 6
+}
+
+InitialOrdosReinforcements =
+{ 
+	{ "trooper", "trooper", "trooper", "trooper", "light_inf", "light_inf", "light_inf" },
+	{ "trooper", "trooper", "trooper", "trooper", "trooper", "combat_tank_o", "combat_tank_o" }
+}
+
+InitialAtreidesReinforcements = { "combat_tank_a", "combat_tank_a", "quad", "quad", "trike" }
+
+InitialMercenaryReinforcements = { "trooper", "trooper", "trooper", "trooper", "quad", "quad" }
+
+OrdosPaths =
+{
+	{ OrdosEntry1.Location, OrdosRally1.Location },
+	{ OrdosEntry2.Location, OrdosRally2.Location },
+	{ OrdosEntry3.Location, OrdosRally3.Location },
+	{ OrdosEntry4.Location, OrdosRally4.Location }
+}
+
+InitialOrdosPaths =
+{
+	{ OLightFactory.Location, OrdosRally5.Location },
+	{ OHiTechFactory.Location, OrdosRally6.Location }
+}
+
+SaboteurPaths =
+{
+	{ SaboteurWaypoint1.Location, SaboteurWaypoint2.Location, SaboteurWaypoint3.Location },
```

Comment:
The mission crashs with the following error message.

```lua
Error message
===============
.../OpenRA//lua/sandbox.lua:150: [string "harkonnen08.lua"]:94: attempt to index global 'SaboteurWaypoint1' (a nil value)
```
It seems that the `SaboteurWaypoints` are missing in the `map.yaml`.